app/shared/directives/noMobileSupport/noMobileSupport.html

```diff
@@ -0,0 +1,7 @@
+<div class="no-mobile-support">
+	<div class="jumbotron">
+		<p class="lead">IPA does not currently support mobile devices.</p>
```

Comment:
Let's go with 'This website' or 'Instructional Planning and Administration' instead of IPA. I'm thinking of the perspective of those who receive e-mails links to IPA for the first time and don't know what it is.